### PR TITLE
add typings for @google-cloud/tasks

### PR DESCRIPTION
**Pull request by ifiokjr**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`

### CODE REVIEW

**Comment by ifiokjr** [comment]
Help... somebody... anybody 😢 - with Simba (lion king) crying over his lost father

**Comment by ifiokjr** [comment]
@uniqueiniquity anything I can do to get this merged?

**Comment by uniqueiniquity** [comment]
@ifiokjr I'm here! I'll get to this as soon as I can.

File: types/google-cloud__tasks/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,1633 @@
+// Type definitions for @google-cloud/tasks 0.2
+// Project: https://github.com/googleapis/nodejs-tasks (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
```
Comment (uniqueiniquity) [inline types/google-cloud__tasks/index.d.ts]: Please remove the parenthesized clause.

File: types/google-cloud__tasks/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,1633 @@
...
+// Definitions by: Ifiok Jr. <https://github.com/ifiokjr>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// TypeScript Version: 2.8
+
+/// <reference types="node" />
+
+// import * as gax from "google-gax";
```
Comment (uniqueiniquity) [inline types/google-cloud__tasks/index.d.ts]: What are these two commented-out lines for?

File: types/google-cloud__tasks/index.d.ts [hunk 3]
```diff
@@ -0,0 +1,1633 @@
...
+}
+
+export interface Location {
+  displayName: string;
+  labels: {
+    [key: string]: string;
+  };
+  locationId: string;
+  metadata: {
+    [key: string]: any;
```
Comment (uniqueiniquity) [inline types/google-cloud__tasks/index.d.ts]: Prefer `object` over `{ [key: string]: any; }`.

File: types/google-cloud__tasks/ts3.1/index.d.ts [hunk 4]
```diff
@@ -0,0 +1,1622 @@
...
+}
+
+export interface Location {
+  displayName: string;
+  labels: {
+    [key: string]: string;
+  };
+  locationId: string;
+  metadata: {
+    [key: string]: any;
```
Comment (uniqueiniquity) [inline types/google-cloud__tasks/ts3.1/index.d.ts]: Prefer `object` over `{ [key: string]: any; }`

**Review by uniqueiniquity — CHANGES_REQUESTED** [review]

**Comment by ifiokjr** [comment]
Thanks, @uniqueiniquity. I've made the suggested changes.

**Review by uniqueiniquity — APPROVED** [review]

**Comment by Flarna** [comment]
Seems this PR causes a fail in node tests, see [CI](https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/485494766?utm_source=github_status&utm_medium=notification) run of #32552

```
Error in google-cloud__tasks
Error: Expected compilerOptions["baseUrl"] === "../"
    at Object.<anonymous> (/home/travis/build/DefinitelyTyped/DefinitelyTyped/node_modules/dtslint/bin/checks.js:72:27)
    at Generator.next (<anonymous>)
    at fulfilled (/home/travis/build/DefinitelyTyped/DefinitelyTyped/node_modules/dtslint/bin/checks.js:4:58)
    at <anonymous>
```

Most likely the same issue as with jsdom.

**Pull request merged**